x/mongo/driver/operation/create_search_indexes.go

```diff
@@ -109,9 +107,15 @@ func (csi *CreateSearchIndexes) Execute(ctx context.Context) error {
 	return driver.Operation{
 		CommandFn:         csi.command,
 		ProcessResponseFn: csi.processResponse,
+		Client:            csi.session,
+		Clock:             csi.clock,
 		CommandMonitor:    csi.monitor,
+		Crypt:             csi.crypt,
 		Database:          csi.database,
 		Deployment:        csi.deployment,
+		Selector:          csi.selector,
+		ServerAPI:         csi.serverAPI,
+		Timeout:           csi.timeout,
```

Comment:
It's peculiar all this data was missing from the operation. Was it just missed in the original implementation?

Reply:
I think it will be better to align with the [implementation](https://github.com/qingyang-hu/mongo-go-driver/blob/godriver3122/x/mongo/driver/operation/create_indexes.go#L107-L122) in "create_indexes.go".

Reply:
IIUC, the search index management server commands actually manage resources on an Apache Lucene cluster (and maybe some database configuration), so it's possible some of the previously omitted info was unnecessary. However, it's not clear what the impact of adding or removing those fields is.

I generally agree that passing the same info as `CreateIndexes` seems like a good idea, but should we test for that? Or if the info was truly unnecessary, should we continue to omit it?